app/models/proposal.rb

```diff
@@ -45,48 +41,46 @@ def votes_in_tree
     # Rails.cache.fetch("/proposal/#{self.root.id}/votes_in_tree/#{updated_at}", :expires_at => 5.minutes) do
     # Code above seemed to never expire as of Rails 4
     # Proper cache should only cache when votes_in_tree > 100 or so
-    [self.root, self.root.descendants].flatten.map(&:votes_count).sum
+    all_related_proposals.map(&:votes_count).sum
     # end
   end
 
-  # def self_and_descendants    # dupe of all_related_proposals
-  #   [self.root, self.root.descendants].flatten
-  # end
-
   def related_proposals(related_sort_by = 'Most Votes')
```

Comment:
Again, no worries if you wanted to change the API here. If you did, note that the UI is currently broken here per issue #311. Nothing serious, I think just a seam library that changed something.